[docs] Update AABB `get_support` description

beyarkay opened the pull request:
This is a simple documentation edit. It updates the description of the `get_support` method in `AABB` to explain what a support point is.

AThousandShips commented on doc/classes/AABB.xml:
```diff
@@ -126,7 +126,9 @@
 			<description>
-				Returns the support point in a given direction. This is useful for collision detection algorithms.
+				Returns the vertex of the AABB that's the farthest in a given direction.
+				This point is commonly known as the support point in collision detection
+				algorithms.
```
```suggestion
				This point is commonly known as the support point in collision detection algorithms.
```
Keep on one line

aaronfranke commented on doc/classes/AABB.xml:
```diff
@@ -126,7 +126,8 @@
 			<param index="0" name="dir" type="Vector3" />
 			<description>
-				Returns the support point in a given direction. This is useful for collision detection algorithms.
+				Returns the vertex of the AABB that's the farthest in a given direction.
+				This point is commonly known as the support point in collision detection algorithms.
```
```suggestion
				Returns the vertex of the AABB that's the farthest in a given direction. This point is commonly known as the support point in collision detection algorithms.
```
I would go even further and make the entire thing one line.

In the XML docs, every line break is a new paragraph.

beyarkay replied:
Ah my apologies, I didn't realize line breaks cause a new paragraph

clayjohn reviewed (commented):
Looks great! The final step before merging is to squash all the commits together so that the whole PR only contains 1 big commit with all your changes. We like to merge one commit at a time to keep the git history clean and navigable. 

If you don't know how to do that, we have a helpful tutorial in the official documentation https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase

beyarkay committed:
[docs] update AABB `get_support` description

This is a simple documentation edit. It updates the description of the
get_support method in AABB to explain what a support point is.

beyarkay commented:
Apologies for the delay, I've squashed all commits

beyarkay commented:
Just want to check about what the process is here, or what boxes need to be checked before this can be merged? Thanks! I'm sure you're all super busy with the Unity refugees so I really appreciate the time (:

akien-mga commented:
You don't need to do anything further, once approved it's ready to merge by the production team, when we get to it.

akien-mga commented:
Thanks! And congrats for your first merged Godot contribution :tada:
